src/Sarif/ExtensionMethods.cs

```diff
@@ -336,5 +344,105 @@ public static string GetFirstSentence(string text)
             }
             return text.TrimEnd('.') + ".";
         }
+
+        /// <summary>Retrieves a property value if it exists, or null.</summary>
+        /// <param name="properties">A properties object from which the property shall be
+        /// retrieved, or null.</param>
+        /// <param name="key">The property name / key.</param>
+        /// <returns>
+        /// If <paramref name="properties"/> is not null and an entry for the supplied key exists, the
+        /// value associated with that key; otherwise, null.
+        /// </returns>
+        internal static string PropertyValue(this Dictionary<string, string> properties, string key)
```

Comment:
>PropertyValue [](start = 31, length = 13)

Having trouble seeing the value of a wrapper around `TryGetValue`. Perhaps it will be clear when I see it used...